distributed/http/scheduler/api.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import json
+
+from distributed.http.utils import RequestHandler
+
+
+class APIHandler(RequestHandler):
+    def get(self):
+        self.write("API V1")
+        self.set_header("Content-Type", "text/plain")
+
+
+class RetireWorkersHandler(RequestHandler):
+    async def post(self):
+        self.set_header("Content-Type", "text/json")
+        scheduler = self.server
+        try:
+            params = json.loads(self.request.body)
+            workers_info = await scheduler.retire_workers(**params)
+            self.write(json.dumps(workers_info))
+        except Exception as e:
+            self.set_status(400, str(e))
+            self.write(json.dumps({"Error": "Bad request"}))
+
+
+class GetWorkersHandler(RequestHandler):
+    def get(self):
+        scheduler = self.server
+        response = {
+            "num_workers": len(scheduler.workers),
+            "workers": [
+                {"name": ws.name, "address": ws.address}
+                for ws in scheduler.workers.values()
+            ],
+        }
+        self.write(response)
```

Comment:
We should set the `Content-Type` header here to `text/json` and `json.dumps` this dict.